pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -286,36 +288,42 @@ func evictPods(
 				continue
 			}
 
-			if podEvictor.Evict(ctx, pod, evictions.EvictOptions{}) {
-				klog.V(3).InfoS("Evicted pods", "pod", klog.KObj(pod))
-
-				for name := range totalAvailableUsage {
-					if name == v1.ResourcePods {
-						nodeInfo.usage[name].Sub(*resource.NewQuantity(1, resource.DecimalSI))
-						totalAvailableUsage[name].Sub(*resource.NewQuantity(1, resource.DecimalSI))
-					} else {
-						quantity := utils.GetResourceRequestQuantity(pod, name)
-						nodeInfo.usage[name].Sub(quantity)
-						totalAvailableUsage[name].Sub(quantity)
+			shouldEvict, err := podEvictor.PreEvictionFilter(pod)
+			if err != nil {
+				klog.V(1).ErrorS(fmt.Errorf("could not run eviction filter plugin for pod"), "pod", klog.KObj(pod))
```

Comment:
Not sure what I should do here since we are not returning errors in this function. So I am just logging it